politeiawww/backend_test.go

```diff
@@ -0,0 +1,72 @@
+package main
+
+import (
+	"io/ioutil"
+	"os"
+
+	"github.com/stretchr/testify/suite"
+)
+
+const (
+	dataDir = "test_politeiawww_backend"
+)
+
+// ------------------------------------------------------------------------------
```

Comment:
Go doesn't use // ---- comments.  Please don't use these as they may through off godoc.

Reply:
Not necessarily true. I will remove them.